### DISCUSSION TITLE
Add Rounding or "Auto Format" in the interface tab to the Decimal type

### DISCUSSION THREAD

**Discussion by Abdallah-Awwad** [directus/directus#21506] [Feature Requests]
### Summary

When dealing with numbers and calculations you need to store the exact number in the database to make accurate calculations, that's why I chose `Decimal` datatype and Precision & Scale to (10, 5) as I do in other programming languages. 

But the issue that faced me in Directus that I can't change the interface of how I can  show the numbers to the user, because I want to round it to use two digits when it's showing to the user but still be 5 digits in the database. 

### Basic Example

That how it looks when I'm on the record itself:
![image](https://github.com/directus/directus/assets/129249191/949b31ea-d465-46f8-b01e-ec5ca1558383)

Further more, I can see that the `Auto format` is already added in the `Display` tab and it's rounding the number already like this: 
![image](https://github.com/directus/directus/assets/129249191/c42690da-4b1e-44c8-90f0-80c798814395)
So, it does look like this (rounded to two or three digits)
![image](https://github.com/directus/directus/assets/129249191/36a88be4-225c-4616-b048-69ba9d058b64)

But sadly that option doesn't exist in the `interface` which is my case right now. 

### Motivation

It's not familiar to the client to see 5 digits in a number field although it's necessary in some cases to make the calculations accurate so we are trading calculation accuracy for the UI. 

### Detailed Design

It's already implemented in the Display tap, so I hope it won't be hard to apply the same concept in the Interface tab. 

### Must Have:
- Rounding to the nearest value. 

### Could Have:
- Option to choose how many digits to show. 

### Drawbacks

It's a new field, so I don't see any drawbacks (I hope).

### Alternatives

As mentioned above "trading calculation accuracy for the UI", which is unnecessarily specially if it's an easy concept. 

### Adoption Strategy

I think developers who works with some calculations specially the currencies and multiplying and so on, will appreciate this feature because we already doing this on the frontend.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
